community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/compatibility/CompatibilityFor2_2.scala

```diff
@@ -43,7 +43,7 @@ import scala.collection.JavaConverters._
 import scala.util.Try
 
 object helpers {
-  implicit def monitorFailure(t: Throwable)(implicit monitor: QueryExecutionMonitor, session: QuerySession) = {
+  implicit def monitorFailure(t: Throwable)(implicit monitor: QueryExecutionMonitor, session: QuerySession): Unit = {
```

Comment:
Intellij was giving me a warning for having a implicit method without explicit return type.